source/NVDAObjects/IAccessible/chromium.py

```diff
@@ -74,6 +74,15 @@ def _get_states(self):
 		return states
 
 
+class PresentationalList(ia2Web.Ia2Web):
+	""" Ensures that lists like UL, DL and OL always have the readonly state."""

```

Comment:
It might be good to have a note here: 

> work-around for issue #7562 allowing us to differentiate presentational lists from interactive lists (such as of size greater 1 and ARIA list boxes). In firefox, this is possible by the presence of a read-only state, even in content editable.